### DISCUSSION TITLE
Question: Is an edition planned where BthPS3 is merged with DsHidMini?

### DISCUSSION THREAD

**Discussion by SecularSteve** [nefarius/DsHidMini#105] [Q&A] [answered]
If not, is there a reason for that?

**Comment by nefarius** [comment, marked as answer]
No. Both solutions came to life during different epochs of development with different motivations and different backgrounds in the used technologies. It also makes little sense since you still need the USB driver for pairing and general cable use and it would violate my approach of creating individual modular components who have their own dedicated task and do it well.
